### PR TITLE
Cmis write

### PR DESCRIPTION
**Pull request by bwrsandman**
Port of https://code.launchpad.net/~savoirfairelinux-openerp/knowledge-addons/cmis_write/+merge/213940 by @ehdem 

**Depends on #6**

Add cmis_write: It allows to create docs from OE in DMS
## LP reviews
- @max3903: Approve on 2014-04-19
- @bwrsandman: Needs Fixing on 2014-05-06
  
  > No unittests

### CODE REVIEW

File: cmis_write/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+To manage a custom aspect using CMIS (and all the other supported ways)
+you have to:
+
+* Define a new custom model configuring Alfresco. To do this I suggest you
+http://wiki.alfresco.com/wiki/Step-By-Step:_Creating_A_Custom_Model.
+
+* Add the custom aspect to the document you upload or create in Alfresco.
+Using CMIS I suggest you:
+http://docs.alfresco.com/4.1/index.jsp?topic=%2Fcom.alfresco.enterprise.doc%2Fconcepts%2Fopencmis-ext-adding.html.
```
Comment (bwrsandman) [inline cmis_write/__openerp__.py:59]: remove the period at the end of your links, it sends the wrong url

File: cmis_write/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* Uploaded documents will be enqueued for storage in the DMS
+
+Add Metadata
+============
+
+To manage a custom aspect using CMIS (and all the other supported ways)
+you have to:
+
+* Define a new custom model configuring Alfresco. To do this I suggest you
+http://wiki.alfresco.com/wiki/Step-By-Step:_Creating_A_Custom_Model.
```
Comment (bwrsandman) [inline cmis_write/__openerp__.py:55]: remove the period at the end of your links, it sends the wrong url

File: cmis_write/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+* Add the custom aspect to the document you upload or create in Alfresco.
+Using CMIS I suggest you:
+http://docs.alfresco.com/4.1/index.jsp?topic=%2Fcom.alfresco.enterprise.doc%2Fconcepts%2Fopencmis-ext-adding.html.
+
+* Set the custom property in the way you probably know using CMIS.
```
Comment (bwrsandman) [inline cmis_write/__openerp__.py:61]: You may want to reformulate this.

File: cmis_write/tests/test_attachment.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from openerp.tests.common import TransactionCase
+from openerp.addons.connector.session import ConnectorSession
```
Comment (bwrsandman) [inline cmis_write/tests/test_attachment.py:24]: Flake8: `F401 'ConnectorSession' imported but unused`

**Comment by max3903** [comment]
This MP should be in the OCA/connector-cmis, not OCA/knowledge

**Pull request closed without merging**